### DISCUSSION TITLE
Adding support for calculated columns

### DISCUSSION THREAD

**Discussion by JustinOstrowsky** [sql-bi/Bravo#487] [Ideas]
Really enjoying Bravo so far. At least for now i was thinking Tabular Studio and other tools are more complicated than i need.

However where it has come up short for me is with calculated columns.
It would be great to view, and format calculated columns in the same way we can measures.
Is this something that is on the road map?

Trying to decide if I should stick to Bravo or invest the time into Tabular studio.

**Comment by albertospelta** [comment]
@JustinOstrowsky we will consider adding DAX Formatter for calculated columns and calculated tables in a future version, after the release of version 1.0. You can follow this #222 for updates.

**Reply by JustinOstrowsky** [reply]
Great thanks! I'm excited about the future of this tool.
I think for now I am better off getting started with Tabular Studio.
